src/elements/LicenseIconography/LicenseIconography.vue

```diff
@@ -133,11 +117,6 @@
           return stringList.slice(0, max)
         }
       }
-    },
-    methods: {
-      showsPopup: function (index) {
-        return this.computedStrings[index] || this.$slots[index]
```

Comment:
Also delete the prop `stringList` and the computed value `computedStrings`. Basically remove all the redundant code from the component.

Reply:
done